From e38ecc4ec467c5eb1b7e510877532239a43f7db8 Mon Sep 17 00:00:00 2001
From: Viacheslav Bocharov <adeep@lexina.in>
Date: Thu, 6 Jun 2024 14:31:12 +0300
Subject: [PATCH 1/2] dt-bindings: arm: amlogic: add binding for JetHome JetHub
 D2

JetHome JetHub D2 is a home automation controller, based on Amlogic S905X3 SoC

Signed-off-by: Viacheslav Bocharov <adeep@lexina.in>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index a374b98080fe..1acfb184aa9e 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -190,6 +190,7 @@ properties:
               - hardkernel,odroid-c4
               - hardkernel,odroid-hc4
               - haochuangyi,h96-max
+              - jethome,jethub-j200
               - khadas,vim3l
               - libretech,aml-s905d3-cc
               - seirobotics,sei610
-- 
2.45.2

